app/models/gram.rb

```diff
@@ -0,0 +1,53 @@
+class Gram < ActiveRecord::Base
+  # Associations
+  belongs_to :instagram_user
+
+  # Validations
+  validates :ig_id, uniqueness: true
+
+  # Methods
+  def self.collect_latest_posts(user)
+    followees = user.instagram_users
+
+    followees.each do |account|
+      last_post_id = account.grams.last.try :ig_id
+      unless last_post_id == nil
```

Comment:
`unless..else` can lead to booleans being read a little oddly. Considering using `if..else` instead.